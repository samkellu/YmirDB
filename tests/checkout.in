set a 123 33
set b a 123
forward a
forward b
backward a
backward b
snapshot
set c 22 33
del b
snapshot
list entries
del a
list entries
checkout 0
checkout 10
checkout 1
list entries
forward a
forward b
backward a
backward b
bye
