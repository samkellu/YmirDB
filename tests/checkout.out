> ok

> ok

> nil

> a

> b

> nil

> saved as snapshot 1

> ok

> ok

> saved as snapshot 2

> c [22 33]
a [123 33]

> ok

> c [22 33]

> no such snapshot

> no such snapshot

> ok

> b [a 123]
a [123 33]

> nil

> a

> b

> nil

> bye
